cmd/suggestion/nasenvelopenet/main.py

```diff
@@ -0,0 +1,29 @@
+import grpc
+from concurrent import futures
+
+import time
+
+from pkg.api.python import api_pb2_grpc
+from pkg.suggestion.nasenvelopenet_service import EnvelopenetService
+from pkg.suggestion.types import DEFAULT_PORT
+from logging import getLogger, StreamHandler, INFO, DEBUG
+
+
+_ONE_DAY_IN_SECONDS = 60 * 60 * 24
+
+
+def serve():
+    print("NAS Envelopenet Suggestion Service")
```

Comment:
I think, you should change print to logging, like here: https://github.com/kubeflow/katib/blob/master/pkg/suggestion/v1alpha1/nasrl_service.py#L51